packages/batcher/utils/src/config.ts

```diff
@@ -127,7 +127,31 @@ export class ENV {
     return process.env.SELF_SIGNING_API_KEY || '';
   }
 
-  private static isTrue(value: string | undefined): boolean {
+  // MQTT BROKER
+  static get MQTT_BROKER(): boolean {
+    return ENV.isTrue(process.env.MQTT_BROKER, true);
+  }
+  static get MQTT_ENGINE_BROKER_PORT(): number {
+    return parseInt(process.env.MQTT_BROKER_PORT || '8883', 10);
+  }
+  static get MQTT_BATCHER_BROKER_PORT(): number {
+    return parseInt(process.env.MQTT_BROKER_PORT || '8884', 10);
```

Comment:
We have `8883` and `8884` hard-coded in many places despite them being specified here. I think it's because in a lot of places we return `undefined` as a possibility even though these `config` files are meant to avoid the undefined case

Reply:
This is the issue with the 'complex' interface.
This module can be called by paima-engine and the batcher.

They do not share a common interface so one alternative is:

```
import type { ENV as ENV1 } from '@paima/batcher-utils';
import type { ENV as ENV2 } from '@paima/paima-utils';

...
constructor(private env: typeof ENV1 | typeof ENV2) {}
...

env.I_CAN_USED_METHOD_DEFINED_IN_BOTH
```

But this made the Interface having to pass the global ENV (no very nice)